10_code/50_analysis.py

```diff
@@ -0,0 +1,158 @@
+# %%
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from plotnine import *
+import os
+
+# %%
+# Read data
+df = pd.read_csv(
+    "https://raw.githubusercontent.com/MIDS-at-Duke/pds-2022-red-team/main/20_intermediate_files/pop_mortality_merged_no_imputation.csv"
+)
+
+# %%
+# Take a peep:
+df.head()
+
+# %%
+df.isnull().sum()
+
+# %%
+#### post_analysis
+def post_analysis(df, main_state, year_implementation, counter_state):
```

Comment:
Very efficient code! I like how you defined a function rather than repeating the code